apps/saleor-app-checkout/__tests__/pages/api/webhooks/adyen.test.ts

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable require-await */
```

Comment:
Why tho

Reply:
If you think it makes sense, we can disable it for all tests files via `overrides` in eslintconfig

Reply:
Nope, it doesn't make sense, we shouldn't disable this rule - I've replaced the `mockImplementationOnce` with `mockResolvedValueOnce`